tests/disruption_helpers.py

```diff
@@ -67,3 +68,53 @@ def delete_resource(self, resource_id=0):
             condition='Running', selector=self.selector,
             resource_count=self.resource_count, timeout=300
         )
+
+    def kill_daemon(self, node_name=None, check_new_pid=True):
+        """
+        Kill self.resource daemon
+
+        Args:
+            node_name (str): Name of node in which the resource daemon has
+                to be killed
+            check_new_pid (bool): True to check for new pid after killing the
+                daemon. False to skip the check.
+        """
+        node_name = node_name or self.resource_obj[0].pod_data.get('spec').get('nodeName')
+        awk_print = "'{print $1}'"
+        pid_cmd = (
+            f"oc debug node/{node_name} -- chroot /host ps ax | grep"
+            f" ' ceph-{self.resource} --' | grep -v grep | awk {awk_print}"
+        )
+        pid_proc = run_async(pid_cmd)
```

Comment:
run_cmd() or exec_oc_cmd() is not used because it is giving wrong output with this command. shlex.split() in run_cmd is splitting the command in a different way than required.